src/kex_rlwe_latticecrypto/AMD64/ntt_x64.c

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************************
+* LatticeCrypto: an efficient post-quantum Ring-Learning With Errors cryptography library
+*
+*    Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
The library is released under the MIT license (same as OQS). This is noted in the main Readme.md and in src/kex_rlwe_latticecrypto/LICENCE.txt.